src/components/GroupBuildsPage/GroupBuildsPage.tsx

```diff
@@ -1,23 +1,32 @@
-import { Divider, PageSection, PageSectionVariants, Text, TextContent } from '@patternfly/react-core';
-
 import { PageTitles } from 'common/constants';
 
+import { useQueryParamsEffect } from 'hooks/useQueryParamsEffect';
+import { useServiceContainer } from 'hooks/useServiceContainer';
 import { useTitle } from 'hooks/useTitle';
 
-export const GroupBuildsPage = () => {
+import { GroupBuildsList } from 'components/GroupBuildsList/GroupBuildsList';
+import { PageLayout } from 'components/PageLayout/PageLayout';
+
+import * as groupBuildApi from 'services/groupBuildApi';
+
+export const GroupBuildsPage = ({ componentId = 'g1' }) => {
```

Comment:
I am also using `g1` for group config pages, but maybe I need to change to `gc1`, shall we have some naming rules for componentIds?

Reply:
I changed Group Builds page id to `gb1`, please change also Group Configs page id to `gc1` to avoid some initial conflicts.